src/mobile/apps/artwork/components/bid/view.coffee

```diff
@@ -28,8 +30,11 @@ module.exports = class ArtworkBidView extends Backbone.View
     @$el.append modal.$el
     modal.insertModalContent pages.$el
 
-  setupAuctionClock: (sale) ->
-    @auctionClockView = new AuctionClockView
-      model: sale
-      el: @$('.artwork-auction-bid-module__clock')
-    @auctionClockView.start()
+  setupDesktopAuctionClock: (sale) ->
+    if { start_at, end_at, live_start_at } = ARTWORK.auction
```

Comment:
Whoa haven't seen this syntax before! Cool that it works like this.

Reply:
Not that it matters here but one thing to note is that in most linters this sort of things is restricted (no-conditional-assigment) because its a bit hard to unpack at a glance.